gatsby_fp/src/styles/facultyprofile.css

```diff
@@ -99,3 +101,25 @@ hr.top-bar{
     padding: 0;
   }
 }
+
+@media (min-width: 550px){
+  .bioimage{
+    margin-right: 2%;
+    width: 13em;
+  }
+  .flex_box{
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+  }
+  .pf_field_wrapper{
+    flex-grow: 1;
+  }
+}
+
+@media (max-width: 550px){
```

Comment:
When using a `min-width` and a `max-width` media query targeting the same breakpoint, they should be off by one. Otherwise, you end up matching both criteria when the viewport is precisely at the value that they both use. You can subtract 1 from your `max-width` or add 1 to your `min-width` depending on whether you intend for a 550px-wide viewport to use the former or the latter styles.